ci/playbooks/hive_cluster/cluster-claim-create.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Claims a cluster from a the provided pool
+-
+  name: Claim an OpenShift Cluster
+  hosts: localhost
+
+  vars:
+    claim_name: "{{ claimName }}"
+    cluster_pool: "{{ clusterPoolName }}"
+    rtn_kube: "{{ kubePath }}"
+    life_time: "{{ lifetime | default(false) }}"
+
+  roles:
+    - role: oc_cli
+
+  tasks:
+    -
```

Comment:
please use `- name: ` instead of this syntaxe (no newline between `-` and `name`)

Reply:
ack, updated